pkg/agent/manager/cache/cache.go

```diff
@@ -113,22 +136,40 @@ type Cache struct {
 	// selectors holds the selector indices, keyed by a selector key
 	selectors map[selector]*selectorIndex
 
-	// staleEntries holds stale registration entries
+	// staleEntries holds stale or new registration entries which require new SVID to be stored in cache
```

Comment:
maybe we should rename the var given its new intention

Reply:
This variable name change would also impact other cache implementations hence I'd be curious to know what @rturner3 and @azdagron think about new variable name.

Reply:
any name changes that would add such complexity could be done as a follow-on PR maybe?